src/EditorFeatures/CSharpTest/ImplementAbstractClass/ImplementAbstractClassTests_ThroughMember.cs

```diff
@@ -0,0 +1,670 @@
+{
+    public abstract int GetOnly { get; }
+    public abstract int SetOnly { set; }
+}
+
+class Derived : Base
+{
+    Base inner;
+
+    public override int GetOnly => inner.GetOnly;
+
+    public override int SetOnly { set => inner.SetOnly = value; }
+}", index: 1, title: string.Format(FeaturesResources.Implement_through_0, "inner"));
+        }
+
+        [Fact, WorkItem(41420, "https://github.com/dotnet/roslyn/issues/41420")]
+        public async Task PropertyWithSingleAccessorIsDelegated_AllOptionsOff()
+        {
+            await TestAllOptionsOffAsync(
+@"abstract class Base
+{
+    public abstract int GetOnly { get; }
+    public abstract int SetOnly { set; }
+}
+
+class [|Derived|] : Base
+{
+    Base inner;
+}",
+@"abstract class Base
+{
+    public abstract int GetOnly { get; }
+    public abstract int SetOnly { set; }
+}
+
+class Derived : Base
+{
+    Base inner;
+
+    public override int GetOnly
+    {
+        get
+        {
+            return inner.GetOnly;
+        }
+    }
+
+    public override int SetOnly
+    {
+        set
+        {
+            inner.SetOnly = value;
+        }
+    }
+}");
+        }
+
+        [Fact, WorkItem(41420, "https://github.com/dotnet/roslyn/issues/41420")]
+        public async Task EventIsDelegated()
+        {
+            await TestInRegularAndScriptAsync(
+@"using System;
+
+abstract class Base
+{
+    public abstract event Action Event;
+}
+
+class [|Derived|] : Base
+{
+    Base inner;
+}",
+@"using System;
+
+abstract class Base
+{
+    public abstract event Action Event;
+}
+
+class Derived : Base
+{
+    Base inner;
+
+    public override event Action Event
+    {
+        add
+        {
+            inner.Event += value;
+        }
+
+        remove
+        {
+            inner.Event -= value;
+        }
+    }
+}", index: 1, title: string.Format(FeaturesResources.Implement_through_0, "inner"));
+        }
+
+        [Fact, WorkItem(41420, "https://github.com/dotnet/roslyn/issues/41420")]
+        public async Task OnlyOverridableMethodsAreOverridden()
+        {
+            await TestInRegularAndScriptAsync(
+@"abstract class Base
+{
+    public abstract void Method();
+
+    public void NonVirtualMethod();
+}
+
+class [|Derived|] : Base
+{
+    Base inner;
+}",
+@"abstract class Base
+{
+    public abstract void Method();
+
+    public void NonVirtualMethod();
+}
+
+class Derived : Base
+{
+    Base inner;
+
+    public override void Method()
+    {
+        inner.Method();
+    }
+}", index: 1, title: string.Format(FeaturesResources.Implement_through_0, "inner"));
+        }
+
+        [Fact, WorkItem(41420, "https://github.com/dotnet/roslyn/issues/41420")]
+        public async Task ProtectedMethodsCannotBeDelegatedThroughBaseType()
+        {
+            await TestExactActionSetOfferedAsync(
+@"abstract class Base
+{
+    protected abstract void Method();
+}
+
+class [|Derived|] : Base
+{
+    Base inner;
+}", new[] { FeaturesResources.Implement_abstract_class });
```

Comment:
Ah you do have a test with all inaccesible members, you can ignore my first comment.